pallets/pallet-asset-switch/src/benchmarking.rs

```diff
@@ -0,0 +1,236 @@
+// KILT Blockchain – https://botlabs.org
+// Copyright (C) 2019-2024 BOTLabs GmbH
+
+// The KILT Blockchain is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// The KILT Blockchain is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+// If you feel like getting in touch with us, you can do so at info@botlabs.org
+
+use xcm::VersionedMultiAsset;
+
+pub struct BenchmarkInfo {
+	pub remote_fee: VersionedMultiAsset,
+}
+
+/// Helper trait implementable by the runtime to set some additional state
+/// before the pallet benchmarks are run.
+///
+/// This is highly dependent on the runtime logic.
+/// If no special conditions are to be met, it can simply be a no-op and return
+/// `None`.
+pub trait BenchmarkHelper {
+	fn setup() -> Option<BenchmarkInfo>;
+}
+
+impl BenchmarkHelper for () {
+	fn setup() -> Option<BenchmarkInfo> {
+		None
+	}
+}
+
+#[frame_benchmarking::v2::instance_benchmarks(where LocalCurrencyBalanceOf<T, I>: Into<u128>)]
```

Comment:
a bit picky but I would prob import it

Reply:
https://github.com/KILTprotocol/kilt-node/pull/671/commits/c6f0b5e8d70c1ded508b7830f924a8f6fe5d9fb7